test/helpers/story/drivers/dapp-story-test-driver.ts

```diff
@@ -132,40 +150,45 @@ export default class DappStoryTestDriver extends StoryTestDriver {
     const dapp = action.actionType
     switch (dapp) {
       case 'UNISWAP': {
-        const newTest = new Test('UNISWAP Swap DAPP', (async () => {
+        const newTest = new Test('UNISWAP Swap DAPP', async () => {
           const borrower = await getNamedSigner('borrower')
           const loan = await LoanStoryTestDriver.getLoan(hre, borrower)
           let shouldPass = true
           //read the state and determine if this should pass
-          if(!loan) shouldPass = false
+          if (!loan) shouldPass = false
           if (shouldPass) {
             await DappStoryTestDriver.swapUniSwap(hre, loan)
           } else {
-            await expect(await DappStoryTestDriver.swapUniSwap(hre, loan)).to.be.reverted
+            await expect(await DappStoryTestDriver.swapUniSwap(hre, loan)).to.be
+              .reverted
           }
-        }))
+        })
         tests.push(newTest)
         break
       }
       case 'SUSHISWAP': {
-        const newTest = new Test('SUSHISWAP Swap DAPP', (async () => {
+        const newTest = new Test('SUSHISWAP Swap DAPP', async () => {
           const borrower = await getNamedSigner('borrower')
           const loan = await LoanStoryTestDriver.getLoan(hre, borrower)
-          let shouldPass = true
+          let shouldPass = false
           //read the state and determine if this should pass
```

Comment:
Thoughts on adding descriptive comments like this over key steps in the story?

Reply:
okay will do